src/placement/tests/jump_map_place_obj.c

```diff
@@ -464,7 +464,7 @@ add_object_class(daos_oclass_id_t cid)
 		return;
 
 	rank_list.rl_nr = NUM_TO_EXTEND;
-	rank_list.rl_ranks = malloc(sizeof(d_rank_t) * NUM_TO_EXTEND);
+	D_ALLOC_ARRAY(rank_list.rl_ranks, NUM_TO_EXTEND);
```

Comment:
This needs to be D_MM_ALLOC_ARRAY() to avoid crashing doesn't it?

Reply:
Yes, but not actual because of user visible. Thanks to point this.